### ISSUE TITLE
Cannot reach the performance of pre-trained model using default code setting

### ISSUE THREAD

**Issue by alex04072000** [chshin10/epinet#10]
Hi,

I found that using the default code and settings. After 6 days of training, the accuracy on validation set is still worse than the provided pre-trained model. Did you exclude some special training settings or strategies in the provided code?

Thank you!

**Comment by mneira10** [comment]
Hi, 

We've also had a lot of trouble reproducing the results presented in the paper. Any updates on this?
